docs/data_sources/eia860.rst

```diff
@@ -122,15 +120,15 @@ the names and descriptions of each of its fields.
 
    * - Data Dictionary
      - Browse Online
-   * - :ref:`generators_eia860`
+   * - :ref: `boiler_generator_assn_eia860`
+     - https://data.catalyst.coop/pudl/boiler_generator_assn_eia860
+   * - :ref: `generators_eia860`
      - https://data.catalyst.coop/pudl/generators_eia860
-   * - :ref:`ownership_eia860`
+   * - :ref: `ownership_eia860`
      - https://data.catalyst.coop/pudl/ownership_eia860
-   * - :ref:`boiler_generator_assn_eia860`
-     - https://data.catalyst.coop/pudl/boiler_generator_assn_eia860
-   * - :ref:`plants_eia860`
+   * - :ref: `plants_eia860`
      - https://data.catalyst.coop/pudl/plants_eia860
-   * - :ref:`utilities_eia860`
+   * - :ref: `utilities_eia860`
```

Comment:
I think the space between `:ref:` and the name of the reference will break these links.